validator/src/main/resources/validations/python/ec2/asg/log-validation.yml

```diff
@@ -3,23 +3,23 @@
   httpPath: "/outgoing-http-call"
   httpMethod: "get"
   callingType: "http"
-  expectedLogStructureTemplate: "PYTHON_EC2_OUTGOING_HTTP_CALL_LOG"
+  expectedLogStructureTemplate: "PYTHON_EC2_ASG_OUTGOING_HTTP_CALL_LOG"
 -
   validationType: "cw-log"
   httpPath: "/aws-sdk-call"
   httpMethod: "get"
   callingType: "http"
-  expectedLogStructureTemplate: "PYTHON_EC2_AWS_SDK_CALL_LOG"
+  expectedLogStructureTemplate: "PYTHON_EC2_ASG_AWS_SDK_CALL_LOG"
 -
   validationType: "cw-log"
   httpPath: "/remote-service"
   httpMethod: "get"
   callingType: "http-with-body"
```

Comment:
This will become `http-with-query` with the update in #65

Reply:
Will update this in the ga-python branch after merge

Reply:
Done